env.yml

```diff
@@ -0,0 +1,40 @@
+name: VPN
+channels:
+  - conda-forge
+dependencies:
+  - clang=14.0.6
+  - clangxx=14.0.6
+  - lld=14.0.6
+  - llvmdev=14.0.6
+  - llvm=14.0.6
+  - python=3.9
+  - nodejs=16.13.*
+  - pip=22.3.1
+  - rust=1.66
+  - go=1.18
+  - compiler-rt
+  - cmake=3.22.1
+  - ninja=1.11.0
+  - pip:
+    - appdirs
```

Comment:
Can we get the pip packages from `requirements.txt` somehow? It would be nice if we don't have to maintain this list twice.

Reply:
It seems we can!